DemoApp/android/build.gradle

```diff
@@ -9,6 +9,7 @@ buildscript {
         ndkVersion = "20.1.5948944"
     }
     repositories {
+        mavenCentral()
         google()
         jcenter()
```

Comment:
It turned out this is not possible just yet, because of some react-native and flipper dependencies. This is also being discussed in:
https://github.com/facebook/yoga/issues/1065
https://github.com/react-native-community/discussions-and-proposals/issues/331

Reply:
Ok. I think it's worth opening a issue in this repo to remind us to remove this when we can. It'll likely stop building once jcenter is fully offline.